site-src/api-types/grpcroute.md

```diff
@@ -1,5 +1,9 @@
 # GRPCRoute
 
+!!! info "Feature State"
+
+    The `GRPCRoute` resource is available in `Alpha` as of `v0.6.0`
```

Comment:
Similar to above, not sure if this will have desired result, but would be great to have different icons/colors for experimental features.

```suggestion
??? example "Experimental Channel in v0.6.0+"

    The `GRPCRoute` resource is Alpha and part of the Experimental Channel in `v0.6.0+`.
```

Reply:
Done